doc/admin.md

```diff
@@ -71,13 +71,13 @@ You should now have terminal access to the Accounting Server.
 
 ## Register the service as a protected resource with the Indigo Identity Access Management (IAM)
 
-1. On the [IAM homepage](https://iam-test.indigo-datacloud.eu/login), click "Self Service Protected Resource Registration".
+1. On the [IAM homepage](https://iam-test.indigo-datacloud.eu/dashboard#/home), click "MitreID Dashboard", then click "Self Service Protected Resource Registration" and then click "New Resource".

```

Comment:
Does Markdown allow you to split this into a bullet-pointed sub-list? I think that would be clearer about the multiple steps if possible.

Reply:
it does :), changed